protocol.html

```diff
@@ -1301,270 +1435,445 @@ <h3 property="schema:name">Internationalization Considerations</h3>
+                    <td property="spec:statement">Amend language and document details.</td>
+                  </tr>
+                  <tr about="#d18392b9-8ffc-4264-a5a2-10e2da6fac32" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-4" rel="spec:changeClass" resource="spec:ChangeClass4">4</a></td>
+                    <td><a href="#server-content-type-includes" rel="spec:changeSubject">#server-content-type-includes</a></td>
+                    <td property="spec:statement">Add requirement for server to include <code>Content-Type</code> in <a href="#server-content-type-includes">messages with content</a>.</td>
+                  </tr>
+                  <tr about="#a91960b6-0543-4acc-8153-689c435c7659" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="#relationship-to-ldp" rel="spec:changeSubject">#relationship-to-ldp</a></td>
+                    <td property="spec:statement">Add section describing the <a href="#relationship-to-ldp">the relationship between Solid Protocol and LDP</a>.</td>
+                  </tr>
+                  <tr about="#b0af4fa6-5d35-47b5-8fe3-ad597dc8bdb5" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="#server-representation-turtle-jsonld" rel="spec:changeSubject">#server-representation-turtle-jsonld</a></td>
+                    <td property="spec:statement">Amend the server requirement emphasising <a href="#server-representation-turtle-jsonld">creating RDF sources</a> and their representations.</td>
+                  </tr>
+                  <tr about="#afd2402a-3174-4f27-af9d-f80461538e95" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="#server-put-patch-uri-assignment" rel="spec:changeSubject">#server-put-patch-uri-assignment</a></td>
+                    <td property="spec:statement">Remove the requirement for implicit and inherited URI assignment after HTTP <code>PUT</code>, <code>PATCH</code> requests that is already defined in referenced specifications.</td>
+                  </tr>
+                  <tr about="#c6129c53-0f96-426f-9b46-49be21197eac" typeof="spec:Change">
+                    <td><a href="https://www.w3.org/Consortium/Process/#class-2" rel="spec:changeClass" resource="spec:ChangeClass2">2</a></td>
+                    <td><a href="#server-slug-uri-assignment" rel="spec:changeSubject">#server-slug-uri-assignment</a></td>
+                    <td property="spec:statement">Remove optional requirement for <code>Slug</code> that is not enhancing the product classes.</td>
```

Comment:
I'm not sure what this removal means (and I'm not sure why it was optional). In my apps, [I'm using POST requests to create containers](https://github.com/NoelDeMartin/soukai-solid/blob/main/src/solid/SolidClient.ts#L253..L264), using the Slug header to control the name of the container. And I assume that it is going to use the Slug for the name of the container. Is that still the case, or was I assuming incorrectly?

Reply:
It was a MAY in https://solidproject.org/TR/2022/protocol-20221231#server-slug-uri-assignment . I've documented the reason for the change in https://github.com/solid/specification/pull/621#remove-server-slug-uri-assignment . If there is new information that should be taken into account towards requiring Slug, we can do that in a new issue. I suggest using PUT (alternatively, PATCH) to create a container.